src/expr/src/vector_op/array.rs

```diff
@@ -17,12 +17,12 @@ use risingwave_common::row::Row;
 use risingwave_common::types::ToOwnedDatum;
 use risingwave_expr_macro::function;
 
-#[function("array(...) -> list")]
+#[function("array(...) -> anyarray", type_infer = "panic")]
```

Comment:
What does it mean by `panic` here? The return type should be inferred somewhere else?

Reply:
Right!